devtools/gradle/src/test/resources/gradle-project/gradle.properties

```diff
@@ -1 +1 @@
-quarkusVersion = 999-SNAPSHOT
\ No newline at end of file
+quarkusVersion = 0.23.2
```

Comment:
This will make it actually resolve 0.23.2 deps, right? We should avoid that. What if we keep it equal to the current Quarkus version and also use the same value in the test? The test could read this property file directly, right?

Reply:
@aloubyansky I am not sure I agree with that. Using the same quarkus version as the current release would require mvn installing the dependent artifacts first, which may not be an ideal approach. Also the dependencies between them may change between releases, which would impact this test.

Reply:
> @aloubyansky I am not sure I agree with that. Using the same quarkus version as the current release would require mvn installing the dependent artifacts first, which may not be an ideal approach.
Actually why not? The dependencies are currently installed before this test runs anyway.

> Also the dependencies between them may change between releases, which would impact this test.
It can easily be adjusted in the test though. OTOH, the advantage of using the current version will be to save build/test time by downloading less artifacts.

What we could do in perspective is create a fake platform for this kind of tests.

Reply:
Perhaps we could merge this solution in order to re-enable the disabled tests and then revisit when this fake platform exists?